bpf/lib/maps.h

```diff
@@ -292,11 +292,12 @@ struct ipcache_key {
 
 /* Global IP -> Identity map for applying egress label-based policy */
 struct bpf_elf_map __section_maps cilium_ipcache = {
-	.type		= BPF_MAP_TYPE_HASH,
+	.type		= LPM_MAP_TYPE,
```

Comment:
A side effect of this will be that we break `cilium bpf ipcache list` I think it's fine but it needs to be stated in the help text of the CLI.